### PR TITLE
GitHub action: version + tag

### PR DESCRIPTION
**Pull request by erin-at-work**
### *Summary*

- Adding a github action to automatically update and tag version from input
- Related: https://github.com/coinbase/coinbase-wallet-sdk/pull/435

See: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

### *How did you test your changes?*

--

### CODE REVIEW

**Comment by erin-at-work** [comment]
Something to also look into: https://github.com/changesets/action

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 22:32:30 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 22:37:24 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 22:55:11 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 23:11:01 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 23:12:06 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 23:14:08 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 23:14:26 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 23:18:22 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by jeongmincho — APPROVED** [review]
last attempt...

**Comment by cb-heimdall** [comment]
Review Error for jeongmincho @ 2022-04-04 23:49:56 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

File: .github/workflows/versioning.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Version tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      packageVersion:
+        description: "The version to publish in MAJOR.MINOR.PATCH format"
+        required: true
+        default: ""
```
Comment (ljharb) [inline .github/workflows/versioning.yml:9]:
```suggestion
```
since it's required, i'd think we don't want a default?

**Review by ljharb — APPROVED** [review]

**Pull request merged**